dotnet/src/SemanticKernel.Core/Functions/NativeFunction.cs

```diff
@@ -293,12 +310,16 @@ private void ThrowNotSemantic()
             }
         }
 
-        string? description = method.GetCustomAttribute<DescriptionAttribute>(inherit: true)?.Description;
+        string? methodDescription = method.GetCustomAttribute<DescriptionAttribute>(inherit: true)?.Description;
```

Comment:
nit: Consider renaming this variable to `functionDescription` to align it with the one above, `functionName`.